src/app/scripts/cac/home/cac-home-templates.js

```diff
@@ -0,0 +1,29 @@
+CAC.Home.Templates = (function (Handlebars) {
+    'use strict';
+
+    var module = {
+        destinations: destinations
+    };
+
+    return module;
+
+    function destinations(destinations) {
+        var source = [
+            '{{#each destinations}}',
+            '<div class="col-sm-4">',
+            '<a class="block block-destination block-half" href="#">',
+            '<h3 class="destination-name">{{this.name}}</h3>',
+            '<h5 class="destination-address">{{this.address}}</h5>',
+            '<h5 class="destination-address-2">{{this.city}}, {{this.state}} {{this.zip}}</h5>',
+            '<img src="{{#if this.wide_image}}{{ this.wide_image }}{{^}}http://placeholthis.it/300x150{{/if}}"',
```

Comment:
Hm.  I'd copied this from the itinerary list template, so I guess that one needs to be fixed, too.